zenko-2018-07-08-json.api.json

```diff
@@ -0,0 +1,379 @@
+                    }
+                }
+            },
+            "output": {
+                "type": "structure",
+                "members": {
+                    "IsTruncated": {
+                        "type": "boolean"
+                    },
+                    "NextMarker": {
+                        "type": "string"
+                    },
+                    "Versions": {
+                        "type": "list",
+                        "member": {
+                            "shape": "responseShape"
+                        }
+                    }
+                }
+            }
+        },
+        "getFailedObject": {
+            "http": {
+                "method": "GET",
+                "requestUri": "/_/backbeat/api/crr/failed/{Bucket}/{Key}"
+            },
+            "input": {
+                "type": "structure",
+                "required": [
+                    "Bucket",
+                    "Key",
+                    "VersionId"
+                ],
+                "members": {
+                    "Bucket": {
+                        "location": "uri",
+                        "locationName": "Bucket"
+                    },
+                    "Key": {
+                        "location": "uri",
+                        "locationName": "Key"
+                    },
+                    "VersionId": {
+                        "location": "querystring",
+                        "locationName": "versionId"
+                    }
+                }
+            },
+            "output": {
+                "type": "structure",
+                "members": {
+                    "IsTruncated": {
+                        "type": "boolean"
+                    },
+                    "Versions": {
+                        "type": "list",
+                        "member": {
+                            "shape": "responseShape"
+                        }
+                    }
+                }
+            }
+        },
+        "retryFailedObjects": {
+            "http": {
+                "method": "POST",
+                "requestUri": "/_/backbeat/api/crr/failed"
+            },
+            "input": {
+                "type": "structure",
+                "required": [
+                    "Body"
+                ],
+                "members": {
+                    "Body": {
+                        "type": "binary",
+                        "isStreaming": false
+                    }
+                },
+                "payload": "Body"
+            },
+            "output": {
+                "type": "structure",
+                "members": {
+                    "Results": {
+                        "type": "list",
+                        "member": {
+                            "shape": "retryResponseShape"
+                        }
+                    }
+                },
+                "payload": "Results"
+            }
+        },
+        "retryFailedObjectsPossibleUpdate": {
```

Comment:
yes, functionally, it is a duplicate of `retryFailedObjects` and unused anywhere in `pensieve-front`, so I believe it can be removed.
They differ only in the call params.

Reply:
Ok I will remove this